packages/esm-patient-notes-app/src/notes/notes-main.component.tsx

```diff
@@ -37,6 +37,10 @@ const NotesMain: React.FC<NotesOverviewProps> = ({ patientUuid, showAddNote, pag
     }
   }, [currentVisit]);
 
+  const sortedNotes = visitNotes?.sort(
```

Comment:
This logic should live [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/f4fe313a56d3526921d2a5725b1863c4dc5d5737/packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts#L54) in the resource file.

Reply:
Both of these have been addressed in the latest commits.

Reply:
Thanks @denniskigen for the fixes